docs/api/closedform_external.md

```diff
@@ -0,0 +1,25 @@
+# Closed Form Models (External)
+
+## Poll–Schumann Model (via `pycontrails`)
+
+The Poll–Schumann model provides a closed-form, physics-based method to estimate aircraft fuel flow and performance along flight trajectories. It is implemented in [`pycontrails`](https://py.contrails.org), an open-source library for modeling contrails and aviation climate impacts.
+
+The model analytically links fuel burn, specific range, and aerodynamic efficiency using parameters such as:
+
+- Aircraft mass and lift-to-drag ratio
+- Thrust setting and Mach number
+- Atmospheric temperature and pressure
+
+The implementation in `pycontrails` (`PollSchumann` class) computes instantaneous fuel flow and total burn over a trajectory segment from inputs like altitude, TAS, Mach, and mass.
+
+!!! note
+    The Poll-Schumann Model requires a full flight trajectory as input for fuel calculations.
+
```

Comment:
It would make sense to directly link to the relevant Python module in a "See Also" admonition:

```
[`pycontrails.module.function`](hyperlink)
```